l10n_fr_state/data/res_country_state_data.yml

```diff
@@ -23,173 +24,102 @@
   Creating a res.country.state record
 - 
   !record {model: res.country.state, id: res_country_state_alsace}:
-    code: A
+    code: 44
```

Comment:
It's the code used by geonames.org. I want to use the same code as geonames.org to be able to use the module "OCA/partner-contact/base_location_geonames_import".